### PR TITLE
BENTO-2378 BENTO-2374

### PR DESCRIPTION
**Pull request by rana22**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

**Comment by Senait7-ux** [comment]
Defaulttabtheme.js  & Tabs.js

I think it would be better to have a way to create a generic and customizable header style instead of having it fixed like & .cases , & .sample and so on.

**Comment by Senait7-ux** [comment]
Tabs.js
I think it would be better to have separate code related to css styling and components.

**Comment by rana22** [comment]
1. Moved the default style to separate file defaultTheme.js
2. We will used index for generic class name but there is also existing feature to add your custom class name (tester may required to have certain class name)

File: packages/tab/src/Tabs.js [hunk 1]
```diff
@@ -1,80 +1,140 @@
...
-    let style = {};
-    if (tabHeaderStyle) {
-      style = tabHeaderStyle.root ? { ...tabHeaderStyle.root } : {};
-      if (currentSelectedTab === index && tabHeaderStyle.selected) {
-        style = { ...style, ...tabHeaderStyle.selected };
-      }
-    }
-    return style;
-  };
+//   /* tab headers and specific style generator */
```
Comment (iksheth) [inline packages/tab/src/Tabs.js:26]: Can we remove Un-used code?

**Review by iksheth — CHANGES_REQUESTED** [review]

File: packages/tab/src/Tabs.js (hunk 1 above)
Comment (rana22) [reply packages/tab/src/Tabs.js:26]: removed unused codes

**Review by iksheth — APPROVED** [review]

**Pull request merged**